src/components/funFacts/FunFactPage.tsx

```diff
@@ -1,10 +1,13 @@
 import React from "react";
+import { funFacts } from "./funfacts"
+import { FunFactComponent } from "./FunFactComponent"
+import "./FunFactPage.scss"
 
 export const FunFactPage: React.FunctionComponent = () => {
   return (
     <main>
-      <h1>FunFactPage</h1>
-      <p>FunFactPage</p>
+      <h1>Fun Facts</h1>
+      <FunFactComponent funFacts = { funFacts } />
```

Comment:
Style is not to have spaces here
```suggestion
      <FunFactComponent funFacts={funFacts} />
```

Reply:
Done, thanks